Gemfile

```diff
@@ -39,15 +39,22 @@ gem 'react-rails'
 # Rexml gem
 gem 'rexml'
 
-# Google Oauth required gems
+# Farady gem
+gem 'faraday'
```

Comment:
I don't think Faraday is needed anymore now that the userinfo endpoint muxing was incorporated into your gem, right?